association/association.py

```diff
@@ -33,8 +44,8 @@ def setActivation(self, runTime):
             num = self.inheritance.getUnitNumber(u)
             self.fsa.halfTurnOnStateFromSpikeSource(generator, self.topology.neuralHierarchyTopology.cells, num)
 
-    def build(self, runTime):
-        self.init()
+    def build(self, runTime,NEAL=None,FSA=None):
```

Comment:
Maybe these should be a separate methods, like useFsa(self, fsa), useNeal(self, neal), build() should really be without any parameters. Once we implement the prime operation, the runTime will go away as well. init method will pick either default fsa,neal or the ones passed in the use functions.